include/deal.II/non_matching/mapping_info.h

```diff
@@ -359,16 +369,19 @@ namespace NonMatching
     const typename Triangulation<dim, spacedim>::cell_iterator &cell,
     const Quadrature<dim> &                                     quadrature)
   {
-    const auto &points  = quadrature.get_points();
-    const auto &weights = quadrature.get_weights();
+    const auto &points = quadrature.get_points();
+    unit_points        = std::vector<Point<dim>>(points.begin(), points.end());
```

Comment:
Isn't it possible to do
```suggestion
    unit_points        = points;
```
since the line above is an `std::vector<Point<dim>>`?